schema.json

```diff
@@ -143,6 +143,15 @@
               "type": "string",
               "description": "e.g. Increased profits by 20% from 2011-2012 through viral advertising"
             }
+          },
+          "keywords": {
+            "type": "array",
+            "description": "Specify technologies or skills used, e.g. HTML, CSS, Javascript",
+            "additionalItems": false,
+            "items": {
+              "type": "string",
+              "description": "e.g. python, leadership, public speaking"
```

Comment:
Maybe give only one example on this line, to make clear that each array item contains only one skill, not one long string containing multiple skills. Just for clarity.